packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -728,7 +728,7 @@ class _CupertinoEdgeShadowDecoration extends Decoration {
   }
 
   @override
-  _CupertinoEdgeShadowPainter createBoxPainter([VoidCallback onChanged]) {
+  _CupertinoEdgeShadowPainter createBoxPainter([ VoidCallback onChanged ]) {
```

Comment:
I'm not sure we want to add spaces for `[`.  I think for `{` it makes sense though.

Reply:
I'm not sure where the space after `{` is coming from in named optional parameters (perhaps @Hixie knows).
I thought the pattern should be similar for optional positional parameters.

Reply:
Let me know what should be done here. I'll make the change if needed.

Reply:
We used to have this in the style guide, but people thought it was micromanaging a bit too much so we've removed it. Do whatever you think is best.

Reply:
I'd just use the same format for named optional parameters and positional optional parameters. I chose to use spaces because it's the most used format across the current codebase. But I'm not sold with it. If no-space looks better I have no problem to use this no-space style (which is what `dartfmt` use).

Reply:
being consistent with dartfmt if the style is as readable either way is probably best to minimize variation.

Reply:
argg, I just merged the PR...

So I can change this style in an other PR to have:

```diff
-void f(int a, int b, { int c = 1, int d }) { ... }
+void f(int a, int b, {int c = 1, int d}) { ... }
-void f(int a, int b, [ int c = 1, int d ]) { ... }
+void f(int a, int b, [int c = 1, int d]) { ... }
```